render_macros/src/children.rs

```diff
@@ -24,16 +24,19 @@ impl Children {
                 quote! { #child }
             })
             .collect();
+
         match children_quotes.len() {
             0 => quote! { Option::<()>::None },
-            1 => quote! { Some(#(#children_quotes)*) },
+            1 => quote! { Some(#(#children_quotes),*) },
             _ => {
                 let mut iter = children_quotes.iter();
+                
                 let first = iter.next().unwrap();
                 let second = iter.next().unwrap();
+
                 let tuple_of_tuples = iter.fold(
                     quote!((#first, #second)),
-                    |renderable, current| quote!((#current, #renderable)),
+                    |renderable, current| quote!((#renderable, #current)),
```

Comment:
Ahhhhhhh I think that's a great catch. It took me a while to grasp the change. Let's debug it in our heads one by one. Imagine `[A, B, C, D]`, then the iterations in `fold` will be:

1. `renderable = A, current = B`
  * `(A, B)`
2. `renderable = (A, B), current = C`
  * `((A, B), C)`
3. `renderable = ((A, B), C), current = D`
  * `(((A, B), C), D)`

I wonder, do you have any cases that this fix does not solve? Or, do you think I understand something wrong? Because it looks like this fixes it 😁

Reply:
It definitely fixes a problem in the simple case (and the test cases prove it), but I have a much deeper / more complex template elsewhere that i'm seeing the nodes re-sort themselves each refresh. Maybe this should be accepted for now, and that looked at as a different issue?